server/service/src/requisition/request_requisition/generate.rs

```diff
@@ -25,9 +25,15 @@ pub fn generate_suggested_quantity(
     }
     let months_of_stock = available_stock_on_hand as f64 / average_monthly_consumption as f64;
 
-    if max_months_of_stock == 0.0
-        || (min_months_of_stock != 0.0 && months_of_stock > min_months_of_stock)
-    {
+    let default_min_months_of_stock: f64;
```

Comment:
I think this is the only case where minstock is called in the back end for calculations, and have changed this to default to the target MOS when it is 0.